external/protobuf/src/google/protobuf/io/tokenizer_unittest.cc

```diff
@@ -672,18 +672,18 @@ DocCommentCase kDocCommentCases[] = {
                               },
                               };
 
-TEST_2D(TokenizerTest, DocComments, kDocCommentCases, kBlockSizes) {
+TEST_2D(TokenizerTest, DocComments, kDocCommentCases, kBlockSizesToken) {
   // Set up the tokenizer.
   TestInputStream input(kDocCommentCases_case.input.data(),
                         kDocCommentCases_case.input.size(),
-                        kBlockSizes_case);
+                        kBlockSizesToken_case);
   TestErrorCollector error_collector;
   Tokenizer tokenizer(&input, &error_collector);
 
   // Set up a second tokenizer where we'll pass all NULLs to NextWithComments().
   TestInputStream input2(kDocCommentCases_case.input.data(),
                         kDocCommentCases_case.input.size(),
-                        kBlockSizes_case);
+                        kBlockSizesToken_case);
```

Comment:
For all above, other than name change there is no logical change done. Is it really required. What is the component/module that depends on this name. I think this is not required

Reply:
In coded_stream_unittest.cc, the same name is defined. It caused a redefinition error.